leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/LeakTraceWordWrapper.kt

```diff
@@ -0,0 +1,368 @@
+package leakcanary.internal.activity.screen
+
+import java.util.Stack
+
+/**
+ * Performs word wrapping of leak traces.
+ */
+object LeakTraceWordWrapper {
+
+  private const val SPACE_CHARACTER = '\u0020'
+  private const val TILDE_CHARACTER = '\u007E'
+  private const val PERIOD_CHARACTER = '\u002E'
+  private const val BREAKING_CHARACTER_LENGTH = 1
+  const val BREAK_CHARACTER = '\n'
```

Comment:
* Consider removing the `_CHARACTER` suffix (this is clear enough)
* `BREAK_CHARACTER` should be private or internal
* Rename `BREAK_CHARACTER` to `LINE_BREAK` ?